express-api/src/services/properties/propertiesServices.ts

```diff
@@ -411,33 +420,37 @@ const makeParcelUpsertObject = async (
       CreatedOn: new Date(),
     });
   }
-
   const classificationId: number = getClassificationOrThrow(row, lookups.classifications);
-
   const adminAreaId: number = getAdministrativeAreaOrThrow(row, lookups.adminAreas);
-
+  const pin = numberOrNull(row.PIN) ?? existentParcel?.PIN;
+  const description = row.Description ?? (existentParcel ? existentParcel.Description : '');
+  const isSensitive = setNewBool(row.IsSensitive, existentParcel.IsSensitive, false);
+  const isVisibleToOtherAgencies = setNewBool(
+    row.IsVisibleToOtherAgencies,
+    existentParcel.IsVisibleToOtherAgencies,
+    true,
+  );
   return {
     Id: existentParcel?.Id,
     AgencyId: getAgencyOrThrowIfMismatched(row, lookups, roles).Id,
     PID: numberOrNull(row.PID),
-    PIN: numberOrNull(row.PIN),
+    PIN: pin,
     ClassificationId: classificationId,
-    Name: row.Name,
-    CreatedById: existentParcel ? undefined : user.Id,
+    Name: row.Name ?? existentParcel.Name ?? '',
+    CreatedById: existentParcel ? existentParcel.CreatedById : user.Id,
     UpdatedById: existentParcel ? user.Id : undefined,
     UpdatedOn: existentParcel ? new Date() : undefined,
-    CreatedOn: existentParcel ? undefined : new Date(),
+    CreatedOn: existentParcel ? existentParcel.CreatedOn : new Date(),
     Location: {
       x: row.Longitude,
       y: row.Latitude,
     },
-    Address1: row.Address,
+    Address1: row.Address ?? existentParcel.Address1,
```

Comment:
if address was not included, and the existentParcel is null, I think this will throw an error.